### PR TITLE
Add dartdoc for tabs and text selection handles

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/scheduler/binding.dart [hunk 1]
```diff
@@ -381,6 +381,19 @@ abstract class SchedulerBinding extends BindingBase {
...
+  /// monotonically increasing. The raw time stamp passed to [handleBeginFrame]
+  /// is monotonic, but the scheduler might adjust those time stamps to provide
+  /// [timeDilation]. Without careful handling, these adjusts could cause time
+  /// to appear to run backwards.
+  ///
+  /// The [resetEpoch] function ensures that the time stamps are monotonic by
+  /// reseting the base time stamp used for future time stamp adjustments to the
+  /// current value. For example, if the [timeDilation] decreases, rather than
+  /// scaling down the [Duration] since the beginning of time, [resetEpoch] will
+  /// ensure that we only scale down the duration since [resetEpoch] was called.
```
Comment (Hixie) [inline packages/flutter/lib/src/scheduler/binding.dart]: this comment is great. One additional thing it should mention is whether to call resetEpoch() before or after setting timeDilation (or if it doesn't matter).

File: packages/flutter/lib/src/widgets/text_selection.dart [hunk 2]
```diff
@@ -21,12 +22,23 @@ import 'overlay.dart';
 /// mixed text: '<the nwor<b quick fox'
 ///   Here 'the b' is selected, but 'brown' is RTL. Both are drawn with the
 ///   [left] type.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/text_selection.dart]: this comment renders... not as intended.
http://docs.flutter.io/flutter/material/TextSelectionHandleType-class.html

File: packages/flutter/lib/src/widgets/text_selection.dart [hunk 3]
```diff
@@ -21,12 +22,23 @@ import 'overlay.dart';
...
+  collapsed,
+}
+
+/// Builds a selection handle of the given type.
 typedef Widget TextSelectionHandleBuilder(BuildContext context, TextSelectionHandleType type);
 
-// Builds a copy/paste toolbar.
+/// Builds a tool bar near a text selection.
+///
+/// Typically displays buttons for copy and pasting text.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/text_selection.dart]: copying and pasting

**Comment by Hixie** [comment]
LGTM

**Pull request merged**